packages/gateway/src/manager.ts

```diff
@@ -50,7 +50,7 @@ export function createGatewayManager(options: CreateGatewayManagerOptions): Gate
     cache: {
       requestMembers: {
         enabled: options.cache?.requestMembers?.enabled ?? false,
-        pending: new Collection(),
+        pending: options.cache?.requestMembers?.pending ?? new Collection(),
```

Comment:
```suggestion
        pending: new Collection(),
```

*Just my opinion*, i don't think having this option has much of a point at all, i known it was already there so would be a breaking change but i don't see the value. If someone does please tell me

Reply:
Yeah, I guessed as much, but I'd rather that deciding to remove this property be left up to somebody with greater contributor experience with the library, perhaps it could be discussed in the dev chat at some point

Reply:
I only bring this up because you made me remember of it, so we will talk for this on discord as it might be better to give this a pr separately